docs/security.rst

```diff
@@ -160,6 +160,26 @@ look like the following.
           -H 'Cache-Control: no-cache' \
           -H "Authorization: Bearer ${ID_TOKEN}"
 
+Basic authentication
+''''''''''''''''''''
+
+Basic username password authentication is currently supported for the API.
```

Comment:
Can you add a link to an external site that will describe this authorization method in detail? 
https://tools.ietf.org/html/rfc7617
https://en.wikipedia.org/wiki/Basic_access_authentication